demo/video_gpuaccel_demo.py

```diff
@@ -0,0 +1,105 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import argparse
+
+import cv2
+import mmcv
+import torch
+import tqdm
+import numpy as np
+from torchvision.transforms import functional as F
+from mmdet.apis import init_detector
+from mmdet.datasets.pipelines import Compose
+
+try:
+    import ffmpegcv
+except ImportError:
+    raise ImportError('Please install ffmpegcv with:\n\n    pip install ffmpegcv')
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='MMDetection video demo')
+    parser.add_argument('video', help='Video file')
+    parser.add_argument('config', help='Config file')
+    parser.add_argument('checkpoint', help='Checkpoint file')
+    parser.add_argument(
+        '--device', default='cuda:0', help='Device used for inference')
+    parser.add_argument(
+        '--score-thr', type=float, default=0.3, help='Bbox score threshold')
+    parser.add_argument('--out', type=str, help='Output video file')
+    parser.add_argument('--show', action='store_true', help='Show video')
+    parser.add_argument('--nvdecode', action='store_true', help='Use NVIDIA decorder')
+    parser.add_argument(
+        '--wait-time',
+        type=float,
+        default=1,
+        help='The interval of show (s), 0 is block')
+    args = parser.parse_args()
+    return args
+
+
+def prefetch_img_metas(model, ori_wh):
+    cfg = model.cfg
+    w, h = ori_wh
+    cfg.data.test.pipeline[0].type = 'LoadImageFromWebcam'
+    test_pipeline = Compose(cfg.data.test.pipeline)
+    data = {'img':np.zeros((h, w, 3), dtype=np.uint8)}
+    data = test_pipeline(data)
+    img_metas = data['img_metas'][0].data
+    return img_metas
+
+
+def process_img(frame_resize, img_metas):
+    assert frame_resize.shape==img_metas['pad_shape']
+    frame_cuda = torch.from_numpy(frame_resize.astype(np.float32)).cuda()
+    frame_cuda = frame_cuda.permute(2, 0, 1) # HWC to CHW
+    mean = torch.from_numpy(img_metas['img_norm_cfg']['mean']).cuda()
```

Comment:
it is my mistake. This value cannot be placed inside.

Reply:
Inside place of `mean` and `std` won't affect the speed. I would like to keep that change.

Reply:
Maybe because the amount of data is too small.  If other people's computers have poor performance, then it should have an impact. It is recommended to remove.